tests/etc/identify_ipf_differences.py

```diff
@@ -1,42 +1,54 @@
+"""Identify differences in Sentinel-1 IPF versions by downloading representative SLCs and extracting support files.
+
+IPF list: https://sar-mpc.eu/processor/ipf/
+"""
+
+import argparse
 import os
 import shutil
-from datetime import datetime
+from dataclasses import dataclass
 from pathlib import Path
-from typing import cast
 from zipfile import ZipFile
 
 import asf_search as asf
 import lxml.etree as ET
-import numpy as np
 
 from burst2safe.utils import get_burst_infos
 
 
+@dataclass
+class Version:
+    id: str
+    important: bool
```

Comment:
What makes a version important?

Reply:
This goes to the general point of the script. If there are changes to the metadata structure for the SAFE (represented by a change in the template xmls), it's an "important" version.